sumcheck/src/lib.rs

```diff
@@ -1,7 +1,8 @@
 #![deny(clippy::cargo)]
+#![feature(decl_macro)]
 #[cfg(feature = "non_pow2_rayon_thread")]
 pub mod local_thread_pool;
-mod macros;
+pub mod macros;
```

Comment:
Now this is explicitly marked as public.  We actually already used these macros across a different crate.